fiftyone/core/dataset.py

```diff
@@ -322,6 +330,7 @@ def __init__(
         self._run_cache = cachetools.LRUCache(5)
 
         self._deleted = False
+        self._write_concern = None
```

Comment:
The idea of globally configuring a dataset's write concern is creative, but it's scary because it's very possible that builtin methods and user code alike will start breaking if one were set to `w=0`, as certain routines may do a sequence of things where some steps implicitly rely on previous steps having been fully completed (ie populating a new field and then immediately computing something about it).

I'd suggest that we not add `Dataset._write_concern` at this time.

Reply:
Sure!